lib/tests/streamlit/server_test.py

```diff
@@ -459,7 +460,8 @@ def test_healthz_with_csrf(self):
 class PortRotateAHundredTest(unittest.TestCase):
     """Tests port rotation handles a MAX_PORT_SEARCH_RETRIES attempts then sys exits"""
 
-    def get_httpserver(self):
+    @staticmethod
+    def get_httpserver():
```

Comment:
and here

Reply:
Meh, it's test code. I'll annotate these, but the purpose of this PR was not to annotate all of `server_test.py`